install these files to a different path to avoid conflicts with gmic itself

--- a/src/gmic_stdlib.gmic
+++ b/src/gmic_stdlib.gmic
@@ -5287,7 +5287,7 @@
   path_test2=${-path_gimp}plug-ins/
   path_test3=${-path_gimp}plug-ins/gmic_gimp_qt/
   if !${-is_windows}
-    path_test4=/usr/share/gmic/
+    path_test4=/usr/share/gmic-krita/
     path_test5=$g_path_unix
   else
     path_test4=$g_path_unix
--- a/gmic-qt/src/Host/KritaPlugin/gmicqttoolplugin.cpp
+++ b/gmic-qt/src/Host/KritaPlugin/gmicqttoolplugin.cpp
@@ -66,7 +66,7 @@
   // Workaround for deploying basic CLUTs
   // See https://krita-artists.org/t/unknown-filename-gmic-qt/37813
   {
-    const auto srcPath = QDir(QCoreApplication::applicationDirPath().append(QStringLiteral("/../share/gmic/"))).absolutePath();
+    const auto srcPath = QDir(QCoreApplication::applicationDirPath().append(QStringLiteral("/../share/gmic-krita/"))).absolutePath();
     const auto dstPath = GmicQt::gmicConfigPath(true);
     const std::list<QString> files = {"/gmic_cluts.gmz", "/gmic_denoise_cnn.gmz"};
 
--- a/gmic-qt/CMakeLists.txt
+++ b/gmic-qt/CMakeLists.txt
@@ -698,8 +698,8 @@
 
 include(GNUInstallDirs)
 
-install(FILES ${CMAKE_SOURCE_DIR}/../resources/gmic_cluts.gmz DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/gmic)
-install(FILES ${CMAKE_SOURCE_DIR}/../resources/gmic_denoise_cnn.gmz DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/gmic)
+install(FILES ${CMAKE_SOURCE_DIR}/../resources/gmic_cluts.gmz DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/gmic-krita)
+install(FILES ${CMAKE_SOURCE_DIR}/../resources/gmic_denoise_cnn.gmz DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/gmic-krita)
 
 set(gmic_qt_QRC
     gmic_qt.qrc
@@ -896,7 +896,7 @@
 else()
 
   message(FATAL_ERROR "GMIC_QT_HOST is not defined as gimp, gimp3, kritaplugin, none, paintdotnet or 8bf")
-  
+
 endif()
 
 feature_summary(WHAT ALL FATAL_ON_MISSING_REQUIRED_PACKAGES)
